src/pages/filter/ui/attribute-header.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactElement } from 'react'
+import cn from 'classnames'
+import Tooltip from 'rc-tooltip'
+
+import { Icon } from '@ui/icon'
+import { FnLabel } from '@components/fn-label'
+import { AttributeKinds, TPropertyStatus } from '@service-providers/common'
+
+interface IAttributeHeader {
+  className?: string
+  attrStatus: TPropertyStatus
+}
+
+// TODO: variants counts (selected/all) from new design
+export const AttributeHeader = ({
+  attrStatus,
+  className,
+}: IAttributeHeader): ReactElement => {
+  const { tooltip, name, title, kind } = attrStatus
+
+  const isFunc = kind === AttributeKinds.FUNC
+
+  return (
+    <div className={cn('flex items-center', className)}>
+      {isFunc && <FnLabel subGroup={true} className="-mr-1" />}
+
+      <span className={cn('text-16 ml-2 font-bold cursor-pointer')}>
+        {name || title}
+      </span>
+
+      {tooltip && (
+        <Tooltip
+          overlay={tooltip}
+          placement="left"
+          trigger={tooltip ? ['click'] : []}
```

Comment:
From line 31 you already know that tooltip not null in this place

Reply:
yep, thx